packages/@aws-cdk/aws-s3-deployment/lib/bucket-deployment.ts

```diff
@@ -183,18 +192,47 @@ export class BucketDeployment extends cdk.Construct {
       throw new Error('Distribution must be specified if distribution paths are specified');
     }
 
+    if (props.enableTempEfsStorage && !props.vpc) {
+      throw new Error('vpc must be specified if enableTempEfsStorage is true');
+    }
+
+    let accessPoint;
+    if (props.enableTempEfsStorage && props.vpc) {
+      const fileSystem = new efs.FileSystem(this, 'deploymentFileSystem', {
+        vpc: props.vpc,
+        removalPolicy: cdk.RemovalPolicy.DESTROY,
+      });
+      accessPoint = fileSystem.addAccessPoint('AccessPoint', {
+        path: '/lambda',
+        createAcl: {
+          ownerUid: '1001',
+          ownerGid: '1001',
+          permissions: '755',
```

Comment:
Is this the least privilege mode we need? Wouldn't 600 work as well?

Reply:
yep, muscle memory fro 755 :) .  I  will test later this week

Reply:
can some from amazon help with getting this new test / snapshot corrected.  
Synthesizing integ.bucket-deployment-cloudfront.js.
Selected stack: test-bucket-deployments-1
Deploying integ.bucket-deployment-cloudfront.js...
test-bucket-deployments-1: deploying...

 ❌  test-bucket-deployments-1 failed: Error: Unable to resolve AWS account to use. It must be either configured when you define your CDK or through the environment

I don't really want to deploy anything here into my own accounts..

Reply:
@bitfrost apologies for the delayed response. The only way to correct the snapshot is to deploy the stack, which is the normal development workflow. I can deploy it for you, but it might prove problematic to iterate on if you don't have a personal environment to inspect and debug. Is this a blocker for you?

Reply:
Sorry for my delayed response.  I just don't have the inclination to setup my personal account to deploy the cdk's test cf stacks into it.  Poking around didn't see clear way to run a minimum subset for this package. I did not look very hard granted).   I don't want to do so in my employer's account either.  Internally we are just using a npm packed version of this package/fork as tgz dependency to get around this issue.

Reply:
> Poking around didn't see clear way to run a minimum subset for this package

Not sure what you mean by that. Looking at the build log i see there are two tests that are failing:

```console
@aws-cdk/aws-s3-deployment: Verifying integ.bucket-deployment-cloudfront.js against integ.bucket-deployment-cloudfront.expected.json ... CHANGED.
@aws-cdk/aws-s3-deployment: Parameters
@aws-cdk/aws-s3-deployment: [~] Parameter AssetParameters8d6188da09b55173c7f04b3f3bc14f28dd68babdd3c0d298b97dc0e1e85bafffS3Bucket61EB803E: {"Type":"String","Description":"S3 bucket for asset \"c24b999656e4fe6c609c31bae56a1cf4717a405619c3aa6ba1bc686b8c2c86cf\""} to {"Type":"String","Description":"S3 bucket for asset \"8d6188da09b55173c7f04b3f3bc14f28dd68babdd3c0d298b97dc0e1e85bafff\""}
@aws-cdk/aws-s3-deployment: [~] Parameter AssetParameters8d6188da09b55173c7f04b3f3bc14f28dd68babdd3c0d298b97dc0e1e85bafffS3VersionKey00CCE044: {"Type":"String","Description":"S3 key for asset version \"c24b999656e4fe6c609c31bae56a1cf4717a405619c3aa6ba1bc686b8c2c86cf\""} to {"Type":"String","Description":"S3 key for asset version \"8d6188da09b55173c7f04b3f3bc14f28dd68babdd3c0d298b97dc0e1e85bafff\""}
@aws-cdk/aws-s3-deployment: [~] Parameter AssetParameters8d6188da09b55173c7f04b3f3bc14f28dd68babdd3c0d298b97dc0e1e85bafffArtifactHash05C6FA49: {"Type":"String","Description":"Artifact hash for asset \"c24b999656e4fe6c609c31bae56a1cf4717a405619c3aa6ba1bc686b8c2c86cf\""} to {"Type":"String","Description":"Artifact hash for asset \"8d6188da09b55173c7f04b3f3bc14f28dd68babdd3c0d298b97dc0e1e85bafff\""}
@aws-cdk/aws-appmesh: [1mtest.gateway-route.js[22m
@aws-cdk/aws-s3-deployment: Verifying integ.bucket-deployment.js against integ.bucket-deployment.expected.json ... CHANGED.
```

So you basically just need to run `yarn integ integ.bucket-deployment-cloudfront.js && yarn integ integ.bucket-deployment.js` in the directory of the package. 

Do you think you'll be able to? If not that i'm afraid we'll have to close this PR out.